docs/tutorials/10_checkpointer_policysaver_tutorial.ipynb

```diff
@@ -373,8 +373,7 @@
         "def train_one_iteration():\n",
         "\n",
         "  # Collect a few steps using collect_policy and save to the replay buffer.\n",
-        "  for _ in range(collect_steps_per_iteration):\n",
```

Comment:
`collect_steps_per_iteration` steps is taken (by DynamicStepDriver) for **each iteration** of this `for loop` (_since, at the time of instantiation, DynamicStepDriver got **num_steps=collect_steps_per_iteration**_).
Hence, at the end of this loop **collect_steps_per_iteration * collect_steps_per_iteration** number of steps will be taken.